Dockerfile

```diff
@@ -78,6 +81,9 @@ COPY composer.json \
     ./
 COPY --from=composer-dev /app/vendor/ vendor/
 
+RUN bin/console assets:install && \
```

Comment:
what was the reason for installing twice, the files from the `prod` target can't get here?

Reply:
Might be different per environment, depending on what’s enabled. Thinking about it, should do it on `dev` as well (and `test` is redundant until there’s a test that goes through a real browser).